FuelSDK-CSharp/FuelSDKConfigurationSection.cs

```diff
@@ -100,6 +100,40 @@ public string Scope
             set { this["scope"] = value; }
 	    }
 
+        /// <summary>
```

Comment:
Add the authorizationCode key in the readme.

Reply:
AuthTokenShouldDifferIfRefreshTokenIsEnforced looks good. It has to have right configurations in the App.config